dkim/verify.go

```diff
@@ -79,6 +83,9 @@ type Verification struct {
 	// The expiration time. If the signature doesn't expire, it's set to zero.
 	Expiration time.Time
 
+	// The QueryResult holds the parsed DNS response
+	QueryResult *queryResult
```

Comment:
We can't add a public field with a private type like this.

Reply:
I don't understand what you mean. Why is it private and why can't we export it?

Reply:
It's unexported, because it begins with a lowercase letter.

Reply:
But `QueryResult` starts with a big letter? Can you explain what you mean? Thanks.

Reply:
The field `QueryResult` is exported, the type `*queryResult` isn't.